### DISCUSSION TITLE
Issue on Github action, when I have more than 4 workflows it stops triggering on sequence

### DISCUSSION THREAD

**Discussion by richiebono** [community/community#29294] [Actions]
Hey Guys, 

I've been doing some tests and found an issue when I call a workflow after workflow using workflow_run.
So I have 6th files that should be called each from 1st until 6th at my repository:

https://github.com/richiebono/codebank-aws/tree/main/.github/workflows

So it should execute these files in this order in a synchronous way:
1-actions-ci-cd-terraform-update.yml
2-actions-ci-cd-transaction-backend.yml
3-actions-ci-cd-invoice-api.yml
4-actions-ci-cd-store-api.yml
5-actions-ci-cd-invoice-frontend.yml
6-actions-ci-cd-store-frontend.yml

I don't know why it's happening, but it just goes until file 4. It just works if I execute some steps unsynchronous.

**Comment by tsaucier-sf** [comment]
I've encountered the same issue, were you able to find a workaround?

**Comment by richiebono** [comment]
In my case, splitting the process into two steps—frontend and backend—worked well because we had only two steps for each application. I created a dependency so that the frontend pipeline only starts when the backend is done.
